src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -459,6 +460,8 @@ class Simpletask : ThemedActivity() {
                 setTitle(R.string.title_filter_drawer)
             }
             Mode.SELECTION -> {
+                val color = getColorDrawable(this, R.attr.colorControlActivated)
```

Comment:
colorControlActivated is the theme color for selected tasks, the selection toolbar, the selection in dialogs, and completed checkboxes. As distinct from colorAccent, which (though it has the same value *right now*) is used for the FAB, the cancel text on dialogs, sliders in the settings (I think), and maybe a couple other things.